### DISCUSSION TITLE
Welcome to openmentoring Discussions!

### DISCUSSION THREAD

**Discussion by jucasoliveira** [ClusterStack/openmentoring#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ricardo-moura** [comment]
@jucasoliveira the word "abroad" should be fixed:

![image](https://user-images.githubusercontent.com/9980738/162511140-bfdc5e1f-e029-429c-abe0-bab1ce668f45.png)

**Reply by jucasoliveira** [reply]
Hey Ricardo, spot on! I'll fix that :)

**Reply by jucasoliveira** [reply]
Fixed!
Next time, feel free to open a branch and create a PR to fix it.
